@navikt/core/tokens/src/shadow.json

```diff
@@ -0,0 +1,38 @@
+{
+  "navds": {
+    "global": {
+      "shadow": {
+        "xs": {
```

Comment:
I systemet ellers bruker vi disse størrelsene: `"2xlarge" | "xlarge" | "large" | "medium" | "small" | "xsmall". Jeg tenker det er fint å være konsekvent på det hele veien.

Reply:
Ok, jeg oppdaterer navnene. Selv synes jeg de er lengre og tyngre enn navnene med to bokstaver.

Reply:
Vi kan godt bytte for min del, så lenge det er konsekvent i hele systemet.